index.bs

```diff
@@ -430,6 +435,15 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+=======
+<h4 id="input-elements">Input Elements</h4>
+
+When an <{input}> element, whose [=browsing context container=]'s [=node document=]'s
+[=origin=] is not [=same origin-domain=] with its [=parent browsing context=]'s document's
+[=origin=], and has its {{type!!attribute}} attribute is in the [=Password state=],
+and is [=focused=], the user agent should round the [=sensor readings=] and reduce the
```

Comment:
I'm curious about what the use cases are for collecting sensor readings when focus in on a nested frame. Shouldn't we consider that background use, and deal with it at a later stage, blocking it altogether normatively for now?